src/libraries/System.Private.CoreLib/src/System/Number.Parsing.cs

```diff
@@ -2081,15 +2081,15 @@ internal enum ParsingStatus
         }
 
         [DoesNotReturn]
-        internal static void ThrowOverflowOrFormatException(ParsingStatus status, TypeCode type = 0) => throw GetException(status, type);
+        internal static void ThrowOverflowOrFormatException(ParsingStatus status, ReadOnlySpan<char> value, TypeCode type = 0) => throw GetException(status, type, value);
 
         [DoesNotReturn]
         internal static void ThrowOverflowException(TypeCode type) => throw GetException(ParsingStatus.Overflow, type);
 
-        private static Exception GetException(ParsingStatus status, TypeCode type)
+        private static Exception GetException(ParsingStatus status, TypeCode type, ReadOnlySpan<char> value = default)
```

Comment:
Why `value = default`?  That suggests there are some call sites that aren't supplying a value, in which case they're going to end up with a strange `The input string '' was not in a correct format.` message.

Reply:
This has now been addressed in my latest commit